deviceinfo.d.ts

```diff
@@ -21,6 +21,7 @@ export function getTimezone(): string;
 export function getInstanceID(): string;
 export function isEmulator(): boolean;
 export function isTablet(): boolean;
+export function getInitialScreenSize(): object;
```

Comment:
Is it possible to define the returned object structure a bit more?
Not a typescript expert, but I think something like this should do the trick:

```typescript
interface IScreenSize {
   width: number;
   height: number;
}

// ...

export function  getInitialScreenSize(): IScreenSize;
```